### PR TITLE
heroku-docker-fix

### PR DESCRIPTION
**Pull request by telegrambotdev**
Fixed Docker runtime errors in heroku

### CODE REVIEW

File: heroku.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: cd /var/www/html/resources && bash heroku-docker-start.sh
```
Comment (Rudloff) [inline heroku.yml:5]: Wouldn't that be enough?
```suggestion
  web: bash /var/www/html/resources/heroku-docker-start.sh
```

File: resources/heroku-docker-start.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+a2dismod mpm_event &&
```
Comment (Rudloff) [inline resources/heroku-docker-start.sh:3]: I don't think `&&` is necessary here.
```suggestion
a2dismod mpm_event
```

**Review by Rudloff — CHANGES_REQUESTED** [review]

File: heroku.yml (hunk 1 above)
Comment (telegrambotdev) [reply heroku.yml:5]: i already tried that.but showed some sort of permission denied error

File: resources/heroku-docker-start.sh (hunk 2 above)
Comment (telegrambotdev) [reply resources/heroku-docker-start.sh:3]: yup.thats right

**Review by telegrambotdev — COMMENTED** [review]
removed &&

**Comment by telegrambotdev** [comment]
suggested changes are done

**Pull request merged**